internal/service/guardduty/detector.go

```diff
@@ -45,85 +46,6 @@ func ResourceDetector() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
-			"datasources": {
-				Type:     schema.TypeList,
-				MaxItems: 1,
-				Optional: true,
-				Computed: true,
```

Comment:
Since this was not previously deprecated, should we deprecate in V6 and slate for removal in V7?

Reply:
Ugh, you're right